galasa-ui/src/components/test-runs/TestRunsTabs.tsx

```diff
@@ -23,14 +27,56 @@ interface TestRunsTabProps {
   resultsNamesPromise: Promise<string[]>;
 }
 
-const TableDesignContent = () => <p>
-    This page is under construction. In future, you will be able to choose which columns are visible and their order.
-</p>;
-
 
 export default function TestRunsTabs({runsListPromise, requestorNamesPromise, resultsNamesPromise}: TestRunsTabProps) {
   const translations = useTranslations("TestRunsTabs");
+  const router = useRouter();
+  const pathname = usePathname();
+  const searchParams = useSearchParams();
+  const [selectedVisibleColumns, setSelectedVisibleColumns] = useState<string[]>(["submittedAt", "testRunName", "requestor", "testName", "status", "result"]);
```

Comment:
Minor - it might be worth having some kind of const or enum that stores the column names so that we don't have to worry about misspelling them whenever we need to use them